.github/workflows/deploy.yml

```diff
@@ -33,11 +30,6 @@
             cat requirements.txt
         - name: deploy DHL
           uses: cloud-gov/cg-cli-tools@main
-          env:
-              CF_API_URL: https://api.fr.cloud.gov
-              CF_SERVICE_USER: ${{secrets.CF_SERVICE_USER}}
-              CF_SERVICE_AUTH: ${{secrets.CF_SERVICE_AUTH}}
-              LM_RUNNER_APP_GUID: ${{secrets.LM_RUNNER_APP_GUID}}
```

Comment:
Any detail about why we can remove these without impact?

Reply:
That was my mistake. I added them in my last DB fix PR and didn't have time to clean them up. The correct place to put them was in `.profile`